ops/README.md

```diff
@@ -197,3 +188,93 @@ Create the `snapshot` and `version` repositories, with:
 The @OTA-Bot GitHub user should have write access to all three (declarations, snapshots, versions) repositories, so it can publish data, create issues, and publish dataset releases.
 
 Each instance should have a responsible entity, which we currently model as a [“team” in the @OpenTermsArchive](https://github.com/orgs/OpenTermsArchive/teams) GitHub organisation. Each team has write access to the three repositories, and @OTA-Bot should be added to that team along with the human maintainers.
+
+## Development
+
+Note that the VM needs to be started before running any commands with `vagrant up`.
+
+If you’re on an Apple Silicon machine or want to use Docker instead of VirtualBox, type `vagrant up --provider=docker`.
+
+### Additional dependencies
+
+To test the changes without impacting the production server, a Vagrantfile is provided to test the changes locally in a virtual machine. 
+[Vagrant](https://www.vagrantup.com/docs/installation/) and [VirtualBox](https://www.virtualbox.org/wiki/Downloads) or [Docker](https://docs.docker.com/get-docker/) are therefore required.
+
+### SSH with vagrant
+
+In order to be able to connect to vagrant through SSH, and deploy with ansible, you need to create a custom SSH key by using:
+
+```
+ssh-keygen -t rsa -f ~/.ssh/ota-vagrant -q -N ""
+```
+
+#### On a Mac with an Apple Silicon processor
+
+VirtualBox is not compatible with Apple Silicon (M1…) processors. You will thus need to use the Docker provider.
+
+**Setup**
+
+Install the `docker` plugin
+
+```
+vagrant plugin install docker
```

Comment:
Did you really have to do that? 🤔 I don't have any plugin installed and it works.

Reply:
@ndpnt checked too and has the same result as me. I took the liberty to remove that instruction, please let me know if you needed it 🙂

Reply:
As I can't remember if it was mandatory (I followed some guidelines on the vagrant page before testing), I'm ok with it being removed